DEVELOPER

```diff
@@ -1,4 +1,3 @@
-
```

Comment:
This file should not have been included in the pull request... oops!

Reply:
Oh no! I included this DEVELOPER file again. I did remove a blank line (eeking emoji here) and that was it.

Reply:
But if the correct choice of action was to remove the DEVELOPER file from the PR my instinct is to do it from my github repo. However, when I look, there doesn't seem to be an option. What to do??